src/init/features/dataconnect/index.ts

```diff
@@ -345,6 +313,46 @@ async function promptForExistingServices(setup: Setup, info: RequiredInfo): Prom
   return info;
 }
 
+interface serviceAndSchema {
+  service: Service;
+  schema?: Schema;
+}
+
+async function chooseExistingService(
+  existing: serviceAndSchema[],
+): Promise<serviceAndSchema | undefined> {
+  const [serviceLocationFromEnvVar, serviceIdFromEnvVar] = serviceEnvVar().split("/");
```

Comment:
I think we should be a bit defensive on this and do some validation

Reply:
https://github.com/firebase/firebase-tools/pull/8677

Discussed offline. Sure, adding some logging here